pkg/beacon/relay/registry/groups_test.go

```diff
@@ -129,7 +129,7 @@ func TestUnregisterStaleGroups(t *testing.T) {
 
 	mockChain.markAsStale(signer2.GroupPublicKeyBytes())
 
-	gr.UnregisterStaleGroups()
+	gr.UnregisterStaleGroups(signer3.GroupPublicKeyBytes())
```

Comment:
Can we add a simple assert `signer3.GroupPublicKeyBytes()` is not checked?

Reply:
Yes, added.